Doc/reference/compound_stmts.rst

```diff
@@ -254,7 +254,8 @@ present, must be last; it matches any exception.  For an except clause with an
 expression, that expression is evaluated, and the clause matches the exception
 if the resulting object is "compatible" with the exception.  An object is
 compatible with an exception if it is the class or a base class of the exception
-object or a tuple containing an item compatible with the exception.
+object, or a tuple containing an item that is the class or a base class of
+the exception object.  (Nested tuples are not allowed.)
```

Comment:
With this reworded sentence, I don't think the parenthesized statement is needed.